test/integration_testcase.go

```diff
@@ -126,6 +128,22 @@ func (tc *IntegrationTestCase) CompareFile(goldenPath, working string) {
 	}
 }
 
+// CompareError compares exected and actual error
+func (tc *IntegrationTestCase) CompareError(err error, stderr string) {
+	wantExists, want := tc.ErrorExpected != "", tc.ErrorExpected
+	gotExists, got := stderr != "" && err != nil, stderr
+
+	if wantExists && gotExists {
+		if !strings.Contains(got, want) {
+			tc.t.Errorf("expected error %s, got error %s", want, got)
```

Comment:
Text here will need to change slightly to match the notion of "contains" instead of "equals"